public/styles/base.css

```diff
@@ -0,0 +1,116 @@
+html {
+  font-family: 'Helvetica';
+  font-size: 1.2em;
+}
+body {
+  background:
+  linear-gradient(
+      rgba(0, 0, 0, 0.1),
+      rgba(0, 0, 0, 0.8)
+    ) no-repeat center center fixed, url(https://images.unsplash.com/photo-1452251889946-8ff5ea7b27ab?ixlib=rb-0.3.5&q=80&fm=jpg&crop=entropy&s=ee0e1e6f2902c4b2338bf54ee31ce2ba) no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+nav[role=navigation] {
```

Comment:
These next few selectors for the nav would be more appropriate in modules for SMACSS